Toolkit/ArcGISToolkit/TemplatePickerViewController.swift

```diff
@@ -179,9 +179,15 @@ public class TemplatePickerViewController: TableViewController {
                     // update info with swatch
                     infos[index].swatch = image
                     
-                    // reload index where that info currently is
                     let indexPath = self.indexPath(for: info)
-                    self.tableView.reloadRows(at: [indexPath], with: .automatic)
+                    let indexPathSection = indexPath.section
+                    let indexPathRow = indexPath.row
```

Comment:
I don't really see a reason for these variables to be declared. Just use `indexPath.section` and `indexPath.row` inline

Reply:
I added the section one because `indexPath.section` was being referred to twice and then I thought if I had a section one there should be a row one. But yeah your right don't really need them.